packages/remix-server-runtime/responses.ts

```diff
@@ -21,17 +21,14 @@ export type DeferFunction = <Data extends Record<string, unknown>>(
   init?: number | ResponseInit
 ) => TypedDeferredData<Data>;
 
-export type JsonFunction = <Data extends unknown>(
```

Comment:
https://typescript-eslint.io/rules/no-unnecessary-type-constraint - `extends unknown` is redundant for these type parameters.

Reply:
The docs are talking about `any` only, but this had `unknown`
Does the same apply?
If so, I guess it would be a good idea to update the docs?

Reply:
Great point 😄 thanks! Filed https://github.com/typescript-eslint/typescript-eslint/issues/6973.